robot/server/server_test.go

```diff
@@ -401,51 +400,6 @@ func TestServer(t *testing.T) {
 		test.That(t, resp.GetMachinePartId(), test.ShouldEqual, "the-robot-part")
 	})
 
-	t.Run("Discovery", func(t *testing.T) {
```

Comment:
I think I will re-add these actually the modmanager doesn't seem to encompass these tests, which is what I originally thought